workflows/rules/results_sharing/upload_to_iva.smk

```diff
@@ -6,12 +6,14 @@ from shutil import copyfile
 
```

Comment:
this is an old rule that isn't used anywhere. we can probably remove it

Reply:
Will fix in separate PR

Reply:
Yeah IVA is short for ingenuity which is the predecessor variant analysis tool to to QCI